app/assets/stylesheets/application.scss

```diff
@@ -65,3 +65,6 @@ $pagination-link-btn-border-color: #ddd;
     }
   }
 }
+.table-center-buttons {
+  text-align: center;
```

Comment:
Use bootstrap's existing classes if possible. For example, here bootstrap already provides text-center class so we don't need to declare our own